apps/frontend-pwa/src/pages/course/[courseId]/element/[id].tsx

```diff
@@ -22,11 +24,23 @@ import {
 import Markdown from '@klicker-uzh/markdown'
 import { addApolloState, initializeApollo } from '@lib/apollo'
 import { getParticipantToken } from '@lib/token'
-import { Button, H3, Progress } from '@uzh-bf/design-system'
+import {
+  Button,
+  H3,
+  H4,
+  Modal,
+  Progress,
+  ThemeContext as Theme,
+  Toast,
+} from '@uzh-bf/design-system'
 import dayjs from 'dayjs'
+import { Form, Formik } from 'formik'
 import { GetServerSideProps } from 'next'
 import { useRouter } from 'next/router'
-import { useEffect, useMemo, useState } from 'react'
+import { useContext, useEffect, useMemo, useState } from 'react'
+import { twMerge } from 'tailwind-merge'
+import * as Yup from 'yup'
+import ContentInput from '../../../../../../frontend-manage/src/components/questions/ContentInput'
```

Comment:
Import from frontend-manage will fail during build / in production

Reply:
component has been moved to shared-components, but replaced by textarea in the end